docs/en/connector-v2/source/pulsar.md

```diff
@@ -0,0 +1,42 @@
+# Apache Pulsar
+
+## Description
+
+Source connector for Apache Pulsar. It can support both off-line and real-time jobs.
+
+##  Options
```

Comment:
Hi, the option format please check other doc, we should display every option in single sub module. After this every connector's doc format will be same.